evennia/help/models.py

```diff
@@ -56,13 +56,23 @@ class HelpEntry(SharedMemoryModel):
     db_key = models.CharField(
         "help key", max_length=255, unique=True, help_text="key to search for"
     )
+
+    @lazy_property
+    def key(self):
+        return self.db_key
+
     # help category
     db_help_category = models.CharField(
         "help category",
         max_length=255,
         default="General",
         help_text="organizes help entries in lists",
     )
+
+    @lazy_property
```

Comment:
Same here, let this be a regular `@property`.

Reply:
Done, changing this property does not cause the unit test to fail.